x/qgb/keeper/keeper.go

```diff
@@ -10,14 +12,16 @@ import (
 )
 
 type Keeper struct {
-	cdc      codec.BinaryCodec
-	storeKey sdk.StoreKey
+	cdc           codec.BinaryCodec
+	storeKey      sdk.StoreKey
+	StakingKeeper *stakingkeeper.Keeper
```

Comment:
its good practice in the sdk to define local interface that only have access to the methods that they need, for instance, in the payment module we have a [`BankKeeper`](https://github.com/celestiaorg/celestia-app/blob/6e0140b9c9434bf8b6a30855bfa884905ae80b9a/x/payment/keeper/keeper.go#L40-L44) to burn funds but it can't send tokens to arbitrary accounts

Reply:
aaaaaah, beautiful stuff. Will change